[llgi] Build failure on arm64-osx

Package: llgi[core,tool]:arm64-osx -> 2023-11-04

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 15.0.0.15000100
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 0e47c1985 2023-12-15 (14 hours ago)

**To Reproduce**

`vcpkg x-set-installed --enforce-port-checks --allow-unsupported llgi[core,tool]`
**Failure logs**

```
-- Using cached altseed-LLGI-1b6b59b9f5bc9f81b4c2af2333d69f6e23670b3e.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/altseed-LLGI-1b6b59b9f5bc9f81b4c2af2333d69f6e23670b3e.tar.gz
-- Applying patch fix-cmake-use-vcpkg.patch
-- Applying patch fix-sources.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean
-- Found external ninja('1.11.1').
-- Configuring arm64-osx
-- Building arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.27.1-osx/cmake-3.27.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/arm64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/install-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/llgi/portfile.cmake:38 (vcpkg_cmake_install)
  scripts/ports.cmake:170 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/install-arm64-osx-dbg-out.log</summary>

```
Change Dir: '/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/arm64-osx-dbg'

Run Build Command(s): /opt/homebrew/bin/ninja -v -v -j11 install
[1/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/LLGI.Texture.cpp.o -MF src/CMakeFiles/LLGI.dir/LLGI.Texture.cpp.o.d -o src/CMakeFiles/LLGI.dir/LLGI.Texture.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/LLGI.Texture.cpp
[2/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/LLGI.CommandList.cpp.o -MF src/CMakeFiles/LLGI.dir/LLGI.CommandList.cpp.o.d -o src/CMakeFiles/LLGI.dir/LLGI.CommandList.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/LLGI.CommandList.cpp
[3/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/LLGI.Graphics.cpp.o -MF src/CMakeFiles/LLGI.dir/LLGI.Graphics.cpp.o.d -o src/CMakeFiles/LLGI.dir/LLGI.Graphics.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/LLGI.Graphics.cpp
[4/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/LLGI.PipelineState.cpp.o -MF src/CMakeFiles/LLGI.dir/LLGI.PipelineState.cpp.o.d -o src/CMakeFiles/LLGI.dir/LLGI.PipelineState.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/LLGI.PipelineState.cpp
[5/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/LLGI.Platform.cpp.o -MF src/CMakeFiles/LLGI.dir/LLGI.Platform.cpp.o.d -o src/CMakeFiles/LLGI.dir/LLGI.Platform.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/LLGI.Platform.cpp
[6/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/PC/LLGI.CreatePC.cpp.o -MF src/CMakeFiles/LLGI.dir/PC/LLGI.CreatePC.cpp.o.d -o src/CMakeFiles/LLGI.dir/PC/LLGI.CreatePC.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/PC/LLGI.CreatePC.cpp
[7/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/LLGI.Buffer.cpp.o -MF src/CMakeFiles/LLGI.dir/LLGI.Buffer.cpp.o.d -o src/CMakeFiles/LLGI.dir/LLGI.Buffer.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/LLGI.Buffer.cpp
[8/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/LLGI.Compiler.cpp.o -MF src/CMakeFiles/LLGI.dir/LLGI.Compiler.cpp.o.d -o src/CMakeFiles/LLGI.dir/LLGI.Compiler.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/LLGI.Compiler.cpp
[9/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/Metal/LLGI.CommandListMetal.mm.o -MF src/CMakeFiles/LLGI.dir/Metal/LLGI.CommandListMetal.mm.o.d -o src/CMakeFiles/LLGI.dir/Metal/LLGI.CommandListMetal.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/Metal/LLGI.CommandListMetal.mm
[10/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/Metal/LLGI.BufferMetal.mm.o -MF src/CMakeFiles/LLGI.dir/Metal/LLGI.BufferMetal.mm.o.d -o src/CMakeFiles/LLGI.dir/Metal/LLGI.BufferMetal.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/Metal/LLGI.BufferMetal.mm
[11/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/Mac/LLGI.WindowMac.mm.o -MF src/CMakeFiles/LLGI.dir/Mac/LLGI.WindowMac.mm.o.d -o src/CMakeFiles/LLGI.dir/Mac/LLGI.WindowMac.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/Mac/LLGI.WindowMac.mm
[12/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/Metal/LLGI.CompilerMetal.mm.o -MF src/CMakeFiles/LLGI.dir/Metal/LLGI.CompilerMetal.mm.o.d -o src/CMakeFiles/LLGI.dir/Metal/LLGI.CompilerMetal.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/Metal/LLGI.CompilerMetal.mm
[13/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/Metal/LLGI.PipelineStateMetal.mm.o -MF src/CMakeFiles/LLGI.dir/Metal/LLGI.PipelineStateMetal.mm.o.d -o src/CMakeFiles/LLGI.dir/Metal/LLGI.PipelineStateMetal.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/Metal/LLGI.PipelineStateMetal.mm
[14/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/Metal/LLGI.RenderPassMetal.mm.o -MF src/CMakeFiles/LLGI.dir/Metal/LLGI.RenderPassMetal.mm.o.d -o src/CMakeFiles/LLGI.dir/Metal/LLGI.RenderPassMetal.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/Metal/LLGI.RenderPassMetal.mm
[15/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/Metal/LLGI.ShaderMetal.mm.o -MF src/CMakeFiles/LLGI.dir/Metal/LLGI.ShaderMetal.mm.o.d -o src/CMakeFiles/LLGI.dir/Metal/LLGI.ShaderMetal.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/Metal/LLGI.ShaderMetal.mm
[16/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_METAL -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/tools/ShaderTranspiler/../ShaderTranspilerCore -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/spirv_cross -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT tools/ShaderTranspiler/CMakeFiles/ShaderTranspiler.dir/main.cpp.o -MF tools/ShaderTranspiler/CMakeFiles/ShaderTranspiler.dir/main.cpp.o.d -o tools/ShaderTranspiler/CMakeFiles/ShaderTranspiler.dir/main.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/tools/ShaderTranspiler/main.cpp
[17/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/Metal/LLGI.GraphicsMetal.mm.o -MF src/CMakeFiles/LLGI.dir/Metal/LLGI.GraphicsMetal.mm.o.d -o src/CMakeFiles/LLGI.dir/Metal/LLGI.GraphicsMetal.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/Metal/LLGI.GraphicsMetal.mm
[18/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/Metal/LLGI.Metal_Impl.mm.o -MF src/CMakeFiles/LLGI.dir/Metal/LLGI.Metal_Impl.mm.o.d -o src/CMakeFiles/LLGI.dir/Metal/LLGI.Metal_Impl.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/Metal/LLGI.Metal_Impl.mm
[19/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/Metal/LLGI.PlatformMetal.mm.o -MF src/CMakeFiles/LLGI.dir/Metal/LLGI.PlatformMetal.mm.o.d -o src/CMakeFiles/LLGI.dir/Metal/LLGI.PlatformMetal.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/Metal/LLGI.PlatformMetal.mm
[20/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/Metal/LLGI.SingleFrameMemoryPoolMetal.mm.o -MF src/CMakeFiles/LLGI.dir/Metal/LLGI.SingleFrameMemoryPoolMetal.mm.o.d -o src/CMakeFiles/LLGI.dir/Metal/LLGI.SingleFrameMemoryPoolMetal.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/Metal/LLGI.SingleFrameMemoryPoolMetal.mm
[21/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_CREATE_COMPILER -DENABLE_METAL  -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT src/CMakeFiles/LLGI.dir/Metal/LLGI.TextureMetal.mm.o -MF src/CMakeFiles/LLGI.dir/Metal/LLGI.TextureMetal.mm.o.d -o src/CMakeFiles/LLGI.dir/Metal/LLGI.TextureMetal.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/src/Metal/LLGI.TextureMetal.mm
[22/26] : && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.27.1-osx/cmake-3.27.1-macos-universal/CMake.app/Contents/bin/cmake -E rm -f src/libLLGI.a && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ar qc src/libLLGI.a  src/CMakeFiles/LLGI.dir/LLGI.Buffer.cpp.o src/CMakeFiles/LLGI.dir/LLGI.CommandList.cpp.o src/CMakeFiles/LLGI.dir/LLGI.Compiler.cpp.o src/CMakeFiles/LLGI.dir/LLGI.Graphics.cpp.o src/CMakeFiles/LLGI.dir/LLGI.PipelineState.cpp.o src/CMakeFiles/LLGI.dir/LLGI.Platform.cpp.o src/CMakeFiles/LLGI.dir/LLGI.Texture.cpp.o src/CMakeFiles/LLGI.dir/PC/LLGI.CreatePC.cpp.o src/CMakeFiles/LLGI.dir/Mac/LLGI.WindowMac.mm.o src/CMakeFiles/LLGI.dir/Metal/LLGI.BufferMetal.mm.o src/CMakeFiles/LLGI.dir/Metal/LLGI.CommandListMetal.mm.o src/CMakeFiles/LLGI.dir/Metal/LLGI.CompilerMetal.mm.o src/CMakeFiles/LLGI.dir/Metal/LLGI.GraphicsMetal.mm.o src/CMakeFiles/LLGI.dir/Metal/LLGI.Metal_Impl.mm.o src/CMakeFiles/LLGI.dir/Metal/LLGI.PipelineStateMetal.mm.o src/CMakeFiles/LLGI.dir/Metal/LLGI.PlatformMetal.mm.o src/CMakeFiles/LLGI.dir/Metal/LLGI.RenderPassMetal.mm.o src/CMakeFiles/LLGI.dir/Metal/LLGI.ShaderMetal.mm.o src/CMakeFiles/LLGI.dir/Metal/LLGI.SingleFrameMemoryPoolMetal.mm.o src/CMakeFiles/LLGI.dir/Metal/LLGI.TextureMetal.mm.o && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib src/libLLGI.a && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.27.1-osx/cmake-3.27.1-macos-universal/CMake.app/Contents/bin/cmake -E touch src/libLLGI.a && :
[23/26] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_METAL -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/spirv_cross -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT tools/ShaderTranspilerCore/CMakeFiles/ShaderTranspilerCore.dir/ShaderTranspilerCore.cpp.o -MF tools/ShaderTranspilerCore/CMakeFiles/ShaderTranspilerCore.dir/ShaderTranspilerCore.cpp.o.d -o tools/ShaderTranspilerCore/CMakeFiles/ShaderTranspilerCore.dir/ShaderTranspilerCore.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/tools/ShaderTranspilerCore/ShaderTranspilerCore.cpp
FAILED: tools/ShaderTranspilerCore/CMakeFiles/ShaderTranspilerCore.dir/ShaderTranspilerCore.cpp.o 
ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DENABLE_METAL -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/spirv_cross -fPIC -g -std=gnu++14 -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.2.sdk -mmacosx-version-min=13.6 -Wall -Werror -MD -MT tools/ShaderTranspilerCore/CMakeFiles/ShaderTranspilerCore.dir/ShaderTranspilerCore.cpp.o -MF tools/ShaderTranspilerCore/CMakeFiles/ShaderTranspilerCore.dir/ShaderTranspilerCore.cpp.o.d -o tools/ShaderTranspilerCore/CMakeFiles/ShaderTranspilerCore.dir/ShaderTranspilerCore.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/tools/ShaderTranspilerCore/ShaderTranspilerCore.cpp
In file included from /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/llgi/src/6e23670b3e-9c2de2829c.clean/tools/ShaderTranspilerCore/ShaderTranspilerCore.cpp:6:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/glslang/SPIRV/GlslangToSpv.h:42:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/glslang/SPIRV/SpvTools.h:50:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/glslang/MachineIndependent/localintermediate.h:42:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/glslang/MachineIndependent/../Include/intermediate.h:57:
In file included from /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/glslang/MachineIndependent/../Include/../Include/Types.h:47:
/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/glslang/MachineIndependent/../Include/SpirvIntrinsics.h:120:10: error: no template named 'variant' in namespace 'std'
    std::variant<const TIntermConstantUnion*, const TType*> value;
    ~~~~~^
1 error generated.
ninja: build stopped: subcommand failed.
```
</details>

--- a/ports/llgi/portfile.cmake
+++ b/ports/llgi/portfile.cmake
@@ -3,8 +3,8 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO altseed/LLGI
-    REF 1b6b59b9f5bc9f81b4c2af2333d69f6e23670b3e
-    SHA512 c9011dee560897caf5ae53d8fa58869b774bd3bc7ce2e0cc4696ac034fc89a36adf3f5285e82cffe6430ca61f6509fd7fbadf5c77aef896c74e8d70e70ff4312
+    REF 8f8510e2dffa1d747ff6ebb0da341198e75291ec
+    SHA512 d521b47f293b90faed28f9648facdfae327c6122ea6391683a08e48558fdf62ce0d3977f78aef3bc276d77ab19fc40ab3cc4d27311dd5a292e0884635fe7c9d3
     HEAD_REF master
     PATCHES
         fix-cmake-use-vcpkg.patch

--- a/ports/llgi/vcpkg.json
+++ b/ports/llgi/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "llgi",
-  "version-date": "2023-11-04",
+  "version-date": "2023-12-19",
   "homepage": "https://github.com/altseed/LLGI",
   "license": null,
   "supports": "!(uwp | android)",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5233,7 +5233,7 @@
       "port-version": 0
     },
     "llgi": {
-      "baseline": "2023-11-04",
+      "baseline": "2023-12-19",
       "port-version": 0
     },
     "llgl": {

--- a/versions/l-/llgi.json
+++ b/versions/l-/llgi.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "70bc60ef323833846200e1ffe0937de2cfaaf77e",
+      "version-date": "2023-12-19",
+      "port-version": 0
+    },
+    {
       "git-tree": "a0f885d3531458ac1a7748ce8af641b30a80a166",
       "version-date": "2023-11-04",
       "port-version": 0

